addons/web_editor/static/lib/odoo-editor/test/spec/html_tables/libre_clipboard.html

```diff
@@ -0,0 +1,87 @@
+        }
+
+        a.comment-indicator:hover+comment {
+            background: #ffd;
+            position: absolute;
+            display: block;
+            border: 1px solid black;
+            padding: 0.5em;
+        }
+
+        a.comment-indicator {
+            background: red;
+            display: inline-block;
+            border: 1px solid black;
+            width: 0.5em;
+            height: 0.5em;
+        }
+
+        comment {
+            display: none;
+        }
+    </style>
+</head>
+
+<body>
+    <table cellspacing="0" border="0">
+        <colgroup width="212"></colgroup>
+        <colgroup width="209"></colgroup>
+        <tr>
+            <td style="border-top: 1px solid #000000; border-bottom: 1px solid #000000; border-left: 1px solid #000000; border-right: 1px solid #000000"
+                height="20" align="left"><i>Italic then also BOLD</i></td>
+            <td style="border-top: 1px solid #000000; border-bottom: 1px solid #000000; border-left: 1px solid #000000; border-right: 1px solid #000000"
+                align="left"><i><s>Italic strike</s></i></td>
+        </tr>
+        <tr>
+            <td style="border-top: 1px solid #000000; border-bottom: 1px solid #000000; border-left: 1px solid #000000; border-right: 1px solid #000000"
+                height="20" align="left"><b>Just bold Just italic</b></td>
+            <td style="border-top: 1px solid #000000; border-bottom: 1px solid #000000; border-left: 1px solid #000000; border-right: 1px solid #000000"
+                align="left"><b><u>Bold underline</u></b></td>
+        </tr>
+        <tr>
+            <td style="border-top: 1px solid #000000; border-bottom: 1px solid #000000; border-left: 1px solid #000000; border-right: 1px solid #000000"
+                height="20" align="left">
+                <font color="#FF0000">Color text</font>
+            </td>
+            <td style="border-top: 1px solid #000000; border-bottom: 1px solid #000000; border-left: 1px solid #000000; border-right: 1px solid #000000"
+                align="left"><u><s>
+                        <font color="#FF0000">Color strike and underline</font>
+                    </s></u></td>
+        </tr>
+        <tr>
+            <td style="border-top: 1px solid #000000; border-bottom: 1px solid #000000; border-left: 1px solid #000000; border-right: 1px solid #000000"
+                height="20" align="left" bgcolor="#FFFF00">Color background</td>
+            <td style="border-top: 1px solid #000000; border-bottom: 1px solid #000000; border-left: 1px solid #000000; border-right: 1px solid #000000"
+                align="left" bgcolor="#FFFF00">
+                <font color="#FF0000">Color text on color background</font>
+            </td>
+        </tr>
+        <tr>
+            <td colspan=2 height="26" align="center" valign=middle>
+                <font face="Andale Mono" size=4>14pt MONO TEXT</font>
+            </td>
+        </tr>
+    </table>
+</body>
+
+</html>
```

Comment:
I don't understand what is `x_clipboard` vs `x_odoo` ? Oh I think I got it. The `x_odoo` one is the result ? Then maybe call it `x_result` ? Sounds weird though but `x_odoo` is _really_ confusing IMO. @Zinston Since you're going to have a quick look at this PR anyway since I pinged you already, do you have a better idea ?

Reply:
_result is fine by me.